tests/credentials_test.go

```diff
@@ -338,25 +345,45 @@ var _ = Describe("[sig-compute]Guest Access Credentials", decorators.SigCompute,
 			}
 			_, err := virtClient.CoreV1().Secrets(vmi.Namespace).Create(context.Background(), &secret, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
+		})
 
+		It("[test_id:TODO]should have ssh-key under authorized keys added by NoCloud", func() {
```

Comment:
You could write these tests as a table, but it can be done in a follow up because there is more to clean up here (e.g. swapping `NewRandomVMIWithEphemeralDiskHighMemory` with `libvmi`).